client/src/components/InboxUserProfilesGroup/InboxUserProfilesGroup.css

```diff
@@ -0,0 +1,18 @@
+.InboxUserProfilesGroup {
+  display: flex;
+  position: relative;
+}
+.InboxUserProfilesGroup__profile-picture:first-child {
+  z-index: 3;
+}
+.InboxUserProfilesGroup__profile-picture:nth-child(2) {
+  position: absolute;
+  left: 2.5rem;
+  z-index: 2;
+}
+.InboxUserProfilesGroup__extra-users,
+.InboxUserProfilesGroup__profile-picture:nth-child(3) {
+  position: absolute;
+  left: 5rem;
+  z-index: 1;
+}
```

Comment:
It looks like this stuff is also in or from #379? Did something happen with your branches maybe? Could we consolidate everything to one branch?

Reply:
Yeah, everything from this branch is merged into component-inbox-message. I thought I needed a separate pull request because it was a separate issue number... oops!